neural_search/util.py

```diff
@@ -43,7 +44,12 @@ def most_similar_by_embedding(
 ) -> List[str]:
     if not is_filter_valid_for_embedding(project_id, embedding_id, att_filter):
         return []
-
+    tmp_limit = limit
+    has_sub_key = embedding.has_sub_key(project_id, embedding_id)
+    if has_sub_key:
+        # new tmp limit to ensure that we get enough results for embedding lists
+        limit_factor = embedding.get_qdrant_limit_factor(project_id, embedding_id)
+        tmp_limit = (limit * limit_factor) + 25
```

Comment:
why +25? seems a bit random

Reply:
you explained that it is a number just to ensure the limit isn't too small; works for me, but then just add a small comment :-)